feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -70,8 +63,21 @@ func (srv *CentralSessionController) CreateSession(
 	ctx context.Context,
 	request *protos.CreateSessionRequest,
 ) (*protos.CreateSessionResponse, error) {
+	glog.V(2).Info("Trying to create session")
+	if err := checkCreateSessionRequest(request); err != nil {
+		return nil, err
+	}
+
+	smPolicyDecision, smPolicyId, err := srv.postSmPoliciesWrapper(request)
+	if err != nil {
+		return nil, err
+	}
+	err = srv.injectOmnipresentRules(smPolicyDecision)
+	if err != nil {
+		glog.Errorf("Failed to inject omnipresent rules")
```

Comment:
Doing this you are hiding the error returned by `injectOmnipresentRules` fooling you when debugging. 

Either 
```
glog.Errorf("Failed to inject omnipresent rules: %s", err)
```
or in this case you don't need to add anything else
glog.Error(err)